codegen/templates/http_client.tmpl

```diff
@@ -126,6 +127,15 @@ func {{$exportName}}(deps *module.Dependencies) Client {
 		}
 	}
 
+	levels := map[string]string{
```

Comment:
It looks like we are not using this eventually to set the different parameters in the circuit breaker, is that part of a follow-up diff?